docs/en-us/dev/user_doc/architecture/design.md

```diff
@@ -187,22 +187,31 @@ Among them, the Master monitors the directories of other Masters and Workers. If
 
 
 
-- Master fault tolerance flowchart：
+- Master fault tolerance：
 
- <p align="center">
-   <img src="https://analysys.github.io/easyscheduler_docs_cn/images/fault-tolerant_master.png" alt="Master fault tolerance flowchart"  width="40%" />
+<p align="center">
+   <img src="/img/failover-master.jpg" alt="failover-master"  width="50%" />
  </p>
```

Comment:
I prefer to use markdown syntax instead of HTML syntax. eg. using `![failover-master](/img/failover-master.jpg)` instead of HTML `p` span

Reply:
We could also keep the old content, in https://github.com/apache/dolphinscheduler/issues/6535 we have and issue to migration html to markdown. We could do this in that time